src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/NewAzureSqlManagedDatabase.cs

```diff
@@ -0,0 +1,211 @@
+        SupportsShouldProcess = true),
+        OutputType(typeof(AzureSqlManagedDatabaseModel))]
+    public class NewAzureSqlManagedDatabase : AzureSqlManagedDatabaseCmdletBase<AzureSqlManagedDatabaseModel>
+    {
+        protected const string CreateNewByNameAndResourceGroupParameterSet =
+            "CreateNewInstanceDatabaseFromInputParameters";
+
+        protected const string CreateNewByInputObjectParameterSet =
+            "CreateNewInstanceDatabaseFromAzureSqlManagedInstanceModelInstanceDefinition";
+
+        protected const string CreateNewByResourceIdParameterSet =
+            "CreateNewInstanceDatabaseFromAzureSqlInstanceResourceId";
+
+        /// <summary>
+        /// Gets or sets the name of the instance database to create.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Instance Database to create.")]
+        [Alias("InstanceDatabaseName")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the instance to use
+        /// </summary>
+        [Parameter(ParameterSetName = CreateNewByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The name of the instance.")]
+        [ValidateNotNullOrEmpty]
+        public override string InstanceName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(ParameterSetName = CreateNewByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 2,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the instance database collation to use
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The collation of the instance database to use.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter("SQL_Latin1_General_CP1_CI_AS", "Latin1_General_100_CS_AS_SC")]
+        public string Collation { get; set; }
+
+        /// <summary>
+        /// Gets or sets the tags associated with the Azure Sql Instance Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The tags to associate with the Azure Sql Instance Database")]
+        [Alias("Tags")]
+        public Hashtable Tag { get; set; }
+
+        /// <summary>
+        /// Gets or sets the instance object
+        /// </summary>
+        [Parameter(ParameterSetName = CreateNewByInputObjectParameterSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Azure Sql Instance object")]
```

Comment:
Notice the inconsistency in help message. This can just be "The instance object".

Reply:
Thanks, changed everywhere